test/manual/amp-analytics/custom-trigger-shadow-2.html

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
```

Comment:
nit: rename the file to have common prefix `analytics-shadow-`? for easier navigation.

Reply:
done